components/Card/Achievement/index.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div id="card">
+    <div class="achievement__main-intro">
+      <p>Kami telah mendapatkan berbagai penghargaan dari:</p>
+      <div class="achievement__main-intro-nav">
+        <CardAchievementSwiper :data="contents" />
+      </div>
+    </div>
+
+    <div class="achievement__main-cards">
+      <swiper
+        ref="mySwiper"
+        :options="swiperOptions"
+      >
+        <swiper-slide
+          v-for="content in contents"
+          :key="content.id"
+        >
+          <div class="achievement__main-cards__card">
+            <div class="achievement__main-cards__card-content">
+              <img
+                width="85"
+                height="85"
+                :src="content.img"
+                :alt="`Achievement ${content.id}`"
+              >
+              <div class="achievement__main-cards__card-content-desc">
+                <div class="achievement__main-cards__card-content-desc-title">
+                  {{ content.title }}
+                </div>
+                <div class="achievement__main-cards__card-content-desc-subtitle">
+                  {{ content.subtitle }}
+                </div>
+                <div class="achievement__main-cards__card-content-desc-text">
+                  {{ content.desc }}
+                </div>
+              </div>
+            </div>
+          </div>
+        </swiper-slide>
+      </swiper>
```

Comment:
ask:

cool swiper @yoslie , is this swiper will be client-side rendered or server-side rendered? 🤔

Reply:
Hmmmm... Good question 🤣 
I wonder about that, but I just read the documentation, there are 2 ways to use it: as client-only component and Directive..
If it's used component method it will be client-side, and if use directive it will render original HTML in server and render Swiper in browser (client)..
I used directive method btw 😅 
reference: https://github.com/surmon-china/surmon-china.github.io/tree/source/projects/vue-awesome-swiper/nuxt

CMIIW @adzharamrullah @Ibwedagama

Reply:
awesome @yoslie , I really need a carousel that can be used with server-side rendering because sometimes we need to server-rendered carousel content for SEO purposes, I will check this out, thank you 🍻